src/content-layout/__tests__/content-layout.test.tsx

```diff
@@ -102,14 +145,129 @@ describe('ContentLayout component', () => {
       });
       expect(isOverlapEnabled()).toBe(true);
     });
+  });
+
+  describe('headerVariant', () => {
+    test('default', () => {
+      const { wrapper } = renderContentLayout({
+        notifications: <>Notifications</>,
+        breadcrumbs: <>Breadcrumbs</>,
+        children: <>Content text</>,
+        header: <>Header text</>,
+        secondaryHeader: <>Secondary text</>,
+      });
+      expect(wrapper.findByClassName(styles.background)!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findHeader()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findNotifications()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findBreadcrumbs()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findSecondaryHeader()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+
+      expect(wrapper.findByClassName(styles['header-wrapper'])!.getElement()).not.toHaveClass(styles['with-divider']);
+    });
+
+    test.skip('high-contrast', () => {
```

Comment:
Is this skipped on purpose? Then it should have a comment

Reply:
Updated tests to cover both themes